enginetest/script_queries.go

```diff
@@ -755,4 +755,156 @@ var ScriptTests = []ScriptTest{
 			},
 		},
 	},
+	{
```

Comment:
You can't put these in here for the same reason you can't change `mytable`: ScriptTests is very generic, meant to support most implementors. Adding the necessity of check constraints breaks that

Put this script into its own var, probably in a new file